src/client.py

```diff
@@ -5,24 +5,38 @@
 sys.path.insert(0, '../generated')
 import tb_controller_pb2
 import tb_controller_pb2_grpc
+import scheduler_pb2
+import scheduler_pb2_grpc
 
-with grpc.insecure_channel('localhost:50053') as channel:
-    stub = tb_controller_pb2_grpc.TieBreaker_ControllerStub(channel)
-    deployment_response = stub.DeployJobMix(tb_controller_pb2.DeploymentRequest(
-        model_types = ['vision', 'vision'],
-        models = ['vgg19', 'mobilenet_v2'],
-        batch_sizes = [2, 4],
-        slos = [1000, 1000],
-        distro_type = 'closed',
-        rps = 100000,
-        slo_percentile = 90
+# A100s - poisson testing SLO violations and actual migration
+with grpc.insecure_channel('10.149.19.24:50053') as channel:
```

Comment:
Let us avoid hard coding IPs. Can this be an argument?

Reply:
Done